### PR TITLE
OCPBUGS-36891: Update OWNERS file in origin/test/extended/router

### PR DESCRIPTION
**Pull request by Thealisyed**
This PR revises:
Updating the OWNERS file in origin/test/extended/router

### CODE REVIEW

File: test/extended/router/OWNERS [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
   - Miciah
   - miheer
+  - alebedev87
+  - candita
+  - gcs278
+  - rfredette
+  - Thealisyed
+  - grzpiotrowski
+emeritus_approvers:
+  - danehans
```
Comment (miheer) [inline test/extended/router/OWNERS:24]: i think we dont this as he is not in the organization

File: test/extended/router/OWNERS [hunk 2]
```diff
@@ -1,14 +1,25 @@
 approvers:
-  - danehans
   - frobware
   - knobunc
   - Miciah
```
Comment (miheer) [inline test/extended/router/OWNERS:4]: Please keep Miciah, miheer and frobware as they in NetEdge team.

**Review by miheer — CHANGES_REQUESTED** [review]

File: test/extended/router/OWNERS (hunk 1 above)
Comment (Thealisyed) [reply test/extended/router/OWNERS:24]: Ok but in the linked jira ticket, a comment was left to move the old owners to emeritus_approvers?

Comment (miheer) [reply test/extended/router/OWNERS:24]: yeah it looks like we need to keep him. No worries.

File: test/extended/router/OWNERS (hunk 2 above)
Comment (miheer) [reply test/extended/router/OWNERS:4]: @Thealisyed i still don't see the above change. Can you please add these 3 names in the approvers and reviewers ?

Comment (miheer) [reply test/extended/router/OWNERS:4]: Not sure why this comment was marked as resolved.

**Comment by miheer** [comment]
@Thealisyed  can you please make the changes I asked for ? I see you marked https://github.com/openshift/origin/pull/29000#discussion_r1718120251 as resolved.

File: test/extended/router/OWNERS (hunk 2 above)
Comment (Thealisyed) [reply test/extended/router/OWNERS:4]: @miheer I can see that Miciah, miheer and frobware are in both approvers and reviewers in the PR?

Comment (candita) [reply test/extended/router/OWNERS:4]: @miheer - I agree, they are in the original file and not removed.

**Pull request merged**